src/mjolnir/valhalla_fetch_transit.cc

```diff
@@ -25,18 +25,33 @@
 #include "midgard/encoded.h"
 #include "midgard/logging.h"
 #include "midgard/sequence.h"
+#include "midgard/tiles.h"
 
 #include "filesystem.h"
 #include "mjolnir/admin.h"
 #include "mjolnir/servicedays.h"
 #include "mjolnir/transitpbf.h"
 #include "mjolnir/util.h"
+#include "third_party/just_gtfs/include/just_gtfs/just_gtfs.h"
 #include "valhalla/proto/transit.pb.h"
 
 using namespace boost::property_tree;
 using namespace valhalla::midgard;
 using namespace valhalla::baldr;
 using namespace valhalla::mjolnir;
+using namespace valhalla;
+using namespace gtfs;
+
+// to include stop_ids or stops
+typedef struct tileTransitInfo {
+  GraphId graphId;
+  Stops tile_stops;
+  Trips tile_trips;
+  Routes tile_routes;
+  std::vector<Id> service_ids;
+  Agencies tile_agencies;
+  int size;
+} tileTransitInfo;
```

Comment:
let's focus on one thing per PR. this PR is only about proving we can read/write with `just_gtfs`. we did that in the tests. after we're happy with that, we'll merge and start the work on "fetching" tiles.